packages/loot-core/src/platform/client/fetch/index-types.ts

```diff
@@ -1,10 +1,10 @@
 import type { Handlers } from '../../../types/handlers';
 import type { ServerEvents } from '../../../types/server-events';
 
-export function init(socketName: string): Promise<unknown>;
+export declare function init(worker: Worker): Promise<unknown>;
```

Comment:
Can we put these types in the existing index.ts?

Reply:
The types are currently shared between the browser and regular index files. We could duplicate them?